Scripts/NuGet/EdFi.RestApi.Databases/Deployment.psm1

```diff
@@ -270,13 +271,33 @@ $deploymentTasks = @{
     }
     'Remove-SandboxDatabases'         = {
         $settings = Get-DeploymentSettings
-        if ($settings.ApiSettings.engine -ne 'SQLServer') { return; }
+
+        if ($settings.ApiSettings.engine -ne 'SQLServer' -and $settings.ApiSettings.engine -ne 'PostgreSQL') { return; }
+
         $masterConnectionStringKey = $settings.ApiSettings.ConnectionStringKeys[$settings.ApiSettings.DatabaseTypes.Master]
         $odsConnectionStringKey = $settings.ApiSettings.ConnectionStringKeys[$settings.ApiSettings.DatabaseTypes.Ods]
 
         $masterCSB = $settings.ApiSettings.csbs[$masterConnectionStringKey]
         $templateCSB = $settings.ApiSettings.csbs[$odsConnectionStringKey]
-        Remove-EdFiSandboxDatabases -masterCSB $masterCSB -edfiOdsTemplateCSB $templateCSB
+
+        if ($settings.ApiSettings.engine -eq 'SQLServer') {
+            Remove-EdFiSandboxDatabases -masterCSB $masterCSB -edfiOdsTemplateCSB $templateCSB
```

Comment:
I think this would be a lot cleaner if this function also took an engine parameter and you had separate functions to remove databases for each engine instead of having all this logic in the deployment script.

Reply:
Done